index.js

```diff
@@ -86,8 +88,66 @@ module.exports = postcss.plugin('postcss-bem-linter', function(primaryOptions, s
       });
     }
 
+    function checkGlob(file, globs) {
+      if (file.indexOf(process.cwd()) === 0) {
+        file = file.substr(process.cwd().length);
+
+        if (file.charAt(0) === path.sep) {
+          file = file.substr(1);
+        }
+      }
+
+      return globs.reduce(function (accumulator, pattern) {
+        return accumulator || minimatch(file, pattern);
+      }, false);
+    }
+
+    function isImplicitComponent(file) {
+      if (config.implicitComponents === true) {
```

Comment:
I removed the `=== true` by removing the first conditional block and simply returning the truthy value at the end of the function.